canonical-web-and-design/deployment-configs
# Add redirects from docs.jujucharms.com to jaas.ai/docs

# Summay

redirect `docs.jujucharms.com` to `jaas.ai/docs`

# QA

- ./qa-deploy --production jaas.ai --staging staging.jaas.ai --tag latest
- ./qa-deploy --production docs.jujucharms.com --staging docs.staging.jujucharms.com --tag latest
- `curl -Is -H 'Host: docs.staging.jujucharms.com' http://127.0.0.1/`
- `curl -Is -H 'Host: docs.staging.jujucharms.com' http://127.0.0.1/blabla`
- Should redirect to staging.jaas.ai/docs
- `curl -Is -H 'Host: docs.jujucharms.com' http://127.0.0.1/`
- `curl -Is -H 'Host: docs.jujucharms.com' http://127.0.0.1/blabla`
- Should redirect to jaas.ai/docs
